docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,53 @@
+Hello Akka Java
```

Comment:
Yes, there are some suggested changes in the Scala seed that I haven't updated here. I wanted the Scala seed to be okayed before applying them here. Will make those changes now.